v2/internal/genericarmclient/cloud_error_details.go

```diff
@@ -0,0 +1,12 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package genericarmclient
+
+type CloudErrorDetails struct {
```

Comment:
Same as below, I'd be OK with this in `core` if you prefer, as it's not a wire-format thing (it's the controllers classification of a wire-format thing). Sorry I wasn't clear before.

Reply:
All good. I've moved `CloudErrorDetails` and `ErrorClassification` back into `core`, leaving the others in `genericarmclient`.